src/main/java/lotto/model/WinningPrice.java

```diff
@@ -0,0 +1,35 @@
+package lotto.model;
+
+public enum WinningPrice {
+    NOTWIN(0),
+    FORTH(5000),
+    THIRD(50000),
+    SECOND(1500000),
+    FIRST(2000000000);
```

Comment:
`_`를 활용해서 가독성을 높일 수 있어요.
```suggestion
    FIRST(2_000_000_000);
```